Apps/Admin/Client/Store/VaccineCard/VaccineCardEffects.cs

```diff
@@ -0,0 +1,91 @@
+//-------------------------------------------------------------------------
+// Copyright © 2019 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-------------------------------------------------------------------------
+
+namespace HealthGateway.Admin.Client.Store.VaccineCard
+{
+    using System;
+    using System.Net.Http;
+    using System.Threading.Tasks;
+    using Fluxor;
+    using HealthGateway.Admin.Client.Api;
+    using HealthGateway.Admin.Client.Store.PatientDetails;
+    using HealthGateway.Admin.Client.Utils;
+    using HealthGateway.Admin.Common.Models.CovidSupport;
+    using HealthGateway.Common.Data.Models;
+    using Microsoft.AspNetCore.Components;
+    using Microsoft.Extensions.Logging;
+    using Refit;
+
+#pragma warning disable CS1591, SA1600
+    public class VaccineCardEffects
+    {
+        public VaccineCardEffects(ILogger<VaccineCardEffects> logger, ISupportApi supportApi, IState<VaccineCardState> vaccineCardState)
+        {
+            this.Logger = logger;
+            this.SupportApi = supportApi;
+            this.VaccineCardState = vaccineCardState;
+        }
+
+        [Inject]
+        private ILogger<VaccineCardEffects> Logger { get; set; }
+
+        [Inject]
+        private ISupportApi SupportApi { get; set; }
+
+        [Inject]
+        private IState<VaccineCardState> VaccineCardState { get; set; }
```

Comment:
`VaccineCardState` doesn't seem to be needed in this class.

Reply:
Removed.